builder.json

```diff
@@ -7,7 +7,7 @@
         "mvn -B compile"
     ],
     "test_steps": [
-        "mvn -B test -DredirectTestOutputToFile=true -DreuseForks=false"
+        "mvn -B test -DredirectTestOutputToFile=true -DforkCount=0 -DrerunFailingTestsCount=5"
```

Comment:
I didn't explicitly decide to change this myself, but took ownership of it from past merges, though I do recognize the motivation for it: if you don't fork the JVM you can attach a debugger to the maven build itself. Otherwise, the maven build always forks and a debugger won't function as you want it to. Also, not forking typically produces more complete output (stdout,stderr,possible logging) from the unit tests which can help push up information for CI builds.

The paired use of rerunFailingTestsCount=5 suggest this is the intent. I'd circle back to Justin to ask if this one is needed. Re-running failed tests by default can be risky and should usually only be done on specific tests with specific care.